### ISSUE TITLE
The indigo adapter must clean up relationships while importing

### ISSUE THREAD

**Issue by longhotsummer** [laws-africa/peachjam#1494]
Suppose we have two works:

* Principal
* Commencement -- which commences Principal

When the Indigo adapter ingests Principal, it creates a relationship between Principal and Commencement. Note that when Commencement is ingested (which could happen before or after), no relationships change. Relationships are created by one side only.

Let's suppose Commencement didn't actually commence Principal. So it is corrected in Indigo and Principal is re-fetched. The old relationship is not removed, and so is still in place, even though it is incorrect.

## Proposal

The Indigo Adapter explicitly works with certain relationships, and it knows which direction they go in. When ingesting a document, it should delete any existing relationships (that it is responsible for) before creating new ones.

In other words, because it knows that Principal should have specific "commenced-by" relationships, it can delete all existing "commenced-by" relationships on Principal before ingesting the new ones. It *cannot* delete *all* relationships, because other works may be responsible for managing those.

Once this is done, all indigo adapters must be refreshed for all sites.
